intelmq/etc/feeds.yaml

```diff
@@ -651,7 +651,7 @@ providers:
         collector:
           module: intelmq.bots.collectors.http.collector_http
           parameters:
-            http_url: https://www.turris.cz/greylist-data/greylist-latest.csv
+            http_url: https://view.sentinel.turris.cz/greylist-data/greylist-latest.csv
```

Comment:
Please update the field `revision` of this entry too

Reply:
Updated.